src/mscorlib/shared/Interop/Unix/Interop.Libraries.cs

```diff
@@ -8,5 +8,6 @@ internal static partial class Libraries
     {
         internal const string GlobalizationInterop = "System.Globalization.Native";
         internal const string SystemNative = "System.Native";
+        internal const string LibCoreCLR = "libcoreclr";
```

Comment:
mscorlib/shared is code shared between CoreCLR and CoreRT. We do not want to have LibCoreCLR defined here because of libcoreclr does not exist in CoreRT.